orbit/cmd/orbit/shell.go

```diff
@@ -98,6 +100,17 @@ var shellCommand = &cli.Command{
 			osquery.WithFlags([]string{"--database_path", osqueryDB}),
 		}
 
+		certPath := filepath.Join(c.String("root-dir"), "certs.pem")
```

Comment:
Double checking:

Will this work if fleetd was built with `fleetctl package [...] --fleet-certificate ...`?

If fleetd was built with `fleetctl package [...] --fleet-certificate ...` then `orbit` starts `osqueryd` with `--tls_server_certs /opt/orbit/fleet.pem`.

One option is to add `--fleet-certificate` as a flag to `shell.go` and if not set then assume `filepath.Join(c.String("root-dir"), "certs.pem")`.